setup.sh

```diff
@@ -311,36 +311,6 @@ elif [[ "$OS_TYPE" == "linux" ]]; then
     fi
 fi
 
-
-
-# Add required PATH exports if not already present
-if ! grep -q "export PATH=\$PATH:/usr/local/go/bin" "$CONFIG_FILE"; then
-    echo "export PATH=\$PATH:/usr/local/go/bin" >> "$CONFIG_FILE"
-fi
-
-if ! grep -q "export PATH=\$HOME/go/bin:\$PATH" "$CONFIG_FILE"; then
-    echo "export PATH=\$HOME/go/bin:\$PATH" >> "$CONFIG_FILE"
```

Comment:
If the corresponding PATH is not defined in $CONFIG_FILE, the trh-sdk binary will not be recognized.